l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -256,24 +119,19 @@ contract L1NativeTokenVault is IL1NativeTokenVault, IL1AssetHandler, Ownable2Ste
         }
     }
 
-    /// @notice Returns the parsed assetId.
-    /// @param _l1TokenAddress The address of the token to be parsed.
-    /// @return The asset ID.
-    function getAssetId(address _l1TokenAddress) public view override returns (bytes32) {
-        return keccak256(abi.encode(block.chainid, L2_NATIVE_TOKEN_VAULT_ADDRESS, _l1TokenAddress));
-    }
-
-    /*//////////////////////////////////////////////////////////////
-                            PAUSE
-    //////////////////////////////////////////////////////////////*/
-
-    /// @notice Pauses all functions marked with the `whenNotPaused` modifier.
-    function pause() external onlyOwner {
-        _pause();
+    // get the computed address before the contract DeployWithCreate2 deployed using Bytecode of contract DeployWithCreate2 and salt specified by the sender
+    function wrappedTokenAddress(
```

Comment:
The name has to remain l2TokenAddress for backward compatibility

Reply:
That's one of a few logic pieces where l2 vs l1 NTV differ. On ethereum vs zksync create2 differ slightly